src/components/Header/Header.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
```

Comment:
app not running error below:
./src/components/Header/Header.tsx
Module not found: Can't resolve '@material-ui/icons/Menu'

maybe you installed the package globally?

Reply:
Review pls..?